webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -171,9 +213,22 @@ const useModalModelCreate = ({ onClose }: Props = {}): ModalHooks => {
             <Form.Item
               label="Model name"
               name="modelName"
-              rules={[{ message: 'Model name is required ', required: true }]}>
+              required
+              requiredMessage="Model name is required ">
               <Input onChange={handleNameChange} />
             </Form.Item>
+            <Tooltip placement="bottom">
+              <Form.Item label="Workspace" name="workspace">
+                <SelectFilter
+                  defaultValue={modalState.workspaceId}
```

Comment:
default value is incorrect 

```ts
Warning: [antd: Form.Item] `defaultValue` will not work on controlled Field. You should use `initialValues` of Form instead. 
```


<img width="727" alt="image" src="https://user-images.githubusercontent.com/109113616/219142153-472757f1-97d1-4c74-ab39-6904e55878df.png">